queries.sql

```diff
@@ -40,3 +40,61 @@ SELECT *
 FROM animals
 WHERE weight>=10.4 and weight<=17.3 
 
+/**ex-2*/
+/*Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.*/
```

Comment:
- [ ] Inside a transaction you should update the animal's table by setting the species column to unspecified, then roll back the change, it may look something like this :point_down: 

```sql
BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; -- verify that change was undone
```